Automattic/wp-calypso
# Plans: Use Selectors instead of lib/plans Helpers

### Testing Instructions

`http://calypso.localhost:3000/plans/`: Upgrade a WP.com site to a higher-tier plan; proceed to checkout, verify that everything still works.
